### ISSUE TITLE
Docking and ConfigFlags

### ISSUE THREAD

**Issue by foulehistory** [ocornut/imgui#6266]
Back-end/Renderer/Compiler/OS

Back-ends: imgui_impl_glfw.cpp + imgui_impl_opengl3.cpp
Compiler: Visual C++ 2022 / VS 17.5.0
Operating System: Windows 11

My Issue/Question:

Hi, I'm using imgui version 1.89.5 WIP, and I wanted to know if ImGuiConfigFlags_NoDockingPreview was not implemented or if it had changed its name? same for ImGuiCol_TabBg.
Because I wanted to change the background color where the tabs were displayed, and not have the arrow and quadrants visible anymore.
Thanks

**Comment by foulehistory** [comment]
Hi, so I want to change the blue color to another and disable the arrow it's possible ?
![image_2023-03-23_173041573](https://user-images.githubusercontent.com/21277960/227271341-4c84c325-ced4-41dc-8970-c5fe3dae1045.png)

and I would like to hide this
![image_2023-03-23_173200333](https://user-images.githubusercontent.com/21277960/227271698-84293479-68ee-4568-9497-5660c859e1bd.png)

thx :)

**Comment by ocornut** [comment]
> Hi, so I want to change the blue color to another a

This is ImGuiCol_TitleBg and variant.

> Disable the arrow it's possible ?

To disable the arrow you would need to get the docking node and set `ImGuiDockNodeFlags_NoWindowMenuButton` in it.
You may also create a ImGuiWindowClass for your window, set `DockNodeFlagsOverrideSet` to `ImGuiDockNodeFlags_NoWindowMenuButton` to call SetNextWindowClass() before Begin() call.
This is all tricky and fragile.

> and I would like to hide this

You cannot, unless you disable docking.

I think it would be better if you explained why you are trying to achieve those things.

**Comment by foulehistory** [comment]
I'm currently trying to make a mini game engine, and I just wanted to hide the squares because I wanted my windows to be anchored on itself (well if we take the squares on the middle one), the arrow I just wanted to hide it too so that we can't interact with it and it's not visible, just to have the tabs visible So I hope I've explained myself better on what I want to do with it

**Comment by ocornut** [comment]
> I just wanted to hide the squares because I wanted my windows to be anchored on itself (well if we take the squares on the middle one), 

I don't understand this sentence about hiding the squares nor I do understand how you expect interactive docking to function without them, it seems senseless and therefore not supported out of the box.
Other things I've answered.

**Comment by foulehistory** [comment]
It's just putting them on transparent so they don't show. I don't see what's crazy about it.

**Comment by foulehistory** [comment]
yes I know, that's why I wanted them to be anchored to each other like on the screen earlier but I'll see thanks.

**Comment by ocornut** [comment]
I don't understand what that means.

**Comment by foulehistory** [comment]
It's okay, I'll find it.
